### PR TITLE
Some more context and comments for translators

### PR DESCRIPTION
**Pull request by Hjdskes**
From #155 (see the comments there).

### CODE REVIEW

**Comment by Hjdskes** [comment]
@piotrdrag can you check the two fuzzy strings in the Polish translation? I did my best to fix the pl.po file but I don't speak Polish..

File: piper/buttondialog.py [hunk 1]
```diff
@@ -220,6 +220,7 @@ def _listbox_filter_func(self, row):
         return True
 
     def _get_button_name_and_description(self, button):
+        # Translators: the {} will be replaced with the button index.
```
Comment (piotrdrag) [inline piper/buttondialog.py]: What is a button index? Its number?

**Comment by piotrdrag** [comment]
Please don’t make any changes to po/pl.po. I will take care of it.

**Comment by Hjdskes** [comment]
> Please don’t make any changes to po/pl.po (other than re-generating it against a new .pot). I will take care of it.

Understood, I'll revert that part.

File: piper/buttondialog.py (hunk 1 above)
Comment (Hjdskes) [inline piper/buttondialog.py]: Yes; each button has an index, e.g. the left mouse button has index 0, the right mouse button has index 1, et cetera. If we map button 1 to button 0, the button with index 0 (left mouse button) will generate the click of button 1 (right mouse button).

Comment (piotrdrag) [inline piper/buttondialog.py]: So how about:
`# Translators: the {} will be replaced with the button index, e.g. "Button 1 click".`

Comment (Hjdskes) [inline piper/buttondialog.py]: Will do, thanks.

**Pull request merged**